src/Create/Create.tsx

```diff
@@ -40,7 +47,7 @@ function JournalEntryForm(){
     
     // Handle form submission
     const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
-        journalEntry.created=new Date().toISOString();
+        journalEntry.created=journalEntry.created || new Date().toISOString();
```

Comment:
I added that because handleSubmit gets called when the journalEntry is created for the first time and also when editing the entry. That's just a check to make sure the original creation date doesn't get overwritten when editing an entry.